spec/folio_client_spec.rb

```diff
@@ -506,11 +505,10 @@
     before do
       stub_request(:post, "#{url}/authn/login")
         .to_return(
-          {status: 200, body: "{\"okapiToken\" : \"#{expired_token}\"}"},
```

Comment:
We already have this stubbed out way above in the top-level `before` block, so we don't need it stubbed again here.